### PR TITLE
Update Getting started page

### PR DESCRIPTION
**Pull request by radinamatic**
### Summary
Just a couple of small improvements.

### Reviewer guidance
Read through and check if everything is OK.

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: docs/getting_started.rst [hunk 1]
```diff
@@ -40,8 +40,6 @@ First, clone your Kolibri fork to your local computer. In command below, replace
 .. code-block:: bash
 
   git clone git@github.com:$USERNAME/kolibri.git
-  cd kolibri  # enter the Kolibri directory
-  git checkout develop
```
Comment (radinamatic) [inline docs/getting_started.rst:44]: It was giving error when run before the `fetch` below.

File: docs/getting_started.rst [hunk 2]
```diff
@@ -137,9 +137,9 @@ The Python project-specific dependencies installed above will install ``nodeenv`
   npm install -g yarn
 
   # other required project dependencies
+  # omit the --force flag to speed up the process
   yarn install --force
 
-Note that the ``--force`` flag above can also be omitted to speed up the process.
```
Comment (radinamatic) [inline docs/getting_started.rst:142]: `--upgrade` flags above did not seem to add too much build time, but this one with `--force` seemed neverending, so I thought it better to add the note _before_ actually running the command.

Comment (indirectlylit) [reply docs/getting_started.rst:142]: I would recommend just replacing the command with `yarn install` and skipping any mention of `--force`. In practice I never do that

File: docs/getting_started.rst [hunk 3]
```diff
@@ -202,7 +201,7 @@ If you need to make the development server available through the LAN, you need t
   # first build the assets
   yarn run build
   # now, run the Django devserver
-  kolibri --debug manage devserver -- 0.0.0.0:8000
```
Comment (radinamatic) [inline docs/getting_started.rst:205]: Must have been a typo! :slightly_smiling_face:

**Comment by radinamatic** [comment]
Comment addressed @indirectlylit!

**Review by indirectlylit — APPROVED** [review]
thanks!

**Pull request merged**